audio/faad2/Portfile

```diff
@@ -15,17 +15,10 @@ long_description \
     Resiliency) object type AAC files.
 homepage            http://www.audiocoding.com/
 platforms           darwin
-master_sites        sourceforge:project/faac/faad2-src/faad2-${version}
-checksums           md5     ee1b4d67ea2d76ee52c5621bc6dbf61e \
-                    sha1    80eaaa5cc576c35dd28863767b795c50cbcc0511 \
-                    rmd160  3f0ca461e2953014d5b17edccb3337767d478b8d
-
-patchfiles          patch-automake-1.13.diff
-
-use_autoreconf  yes
-autoreconf.args -vif
-
-configure.args  --without-xmms --without-drm --without-mpeg4ip
+master_sites        sourceforge:project/faac/faad2-src/faad2-2.8.0
```

Comment:
Instead of hardcoding `2.8.0` here, I'd like to see this computed, e.g. using
```
set branch          [join [lrange [split ${version} .] 0 1] .]
master_sites        sourceforge:project/faac/faad2-src/faad2-${branch}.0
```